apps/api/src/app/auth/services/community.auth.service.ts

```diff
@@ -57,7 +49,7 @@ export class CommunityAuthService implements IAuthService {
     private environmentRepository: EnvironmentRepository,
     private memberRepository: MemberRepository,
     @Inject(forwardRef(() => SwitchOrganization))
```

Comment:
do we still need here to do this inject?

Reply:
unfortunately yes because its circular dependency